src/NexusMods.App.UI/Controls/TopBar/ITopBarViewModel.cs

```diff
@@ -20,4 +21,11 @@ public interface ITopBarViewModel : IViewModelInterface
     public ReactiveCommand<Unit, Unit> MinimizeCommand { get; }
     public ReactiveCommand<Unit, Unit> ToggleMaximizeCommand { get; }
     public ReactiveCommand<Unit, Unit> CloseCommand { get; }
+    public ReactiveCommand<Unit, Unit> HistoryActionCommand { get; }
+    public ReactiveCommand<Unit, Unit> UndoActionCommand { get; }
+    public ReactiveCommand<Unit, Unit> RedoActionCommand { get; }
+    public ReactiveCommand<Unit, Unit> HelpActionCommand { get; }
+    public ReactiveCommand<Unit, Unit> SettingsActionCommand { get; }
```

Comment:
These buttons currently do nothing, and it's unclear if we're even going to use them in the future. It'd be better to not have them in the first place, instead of having a button that does nothing.

Reply:
I talked with Design about this, we decided to have them as disabled for now instead of hidden or not implementing them, so that it is easier to match Design and know where things are/will be when using the App.
Those are likely to be implemented in the future at least.